ui/k-toolbar/main.kytos

```diff
@@ -5,6 +5,19 @@
                           icon="desktop"
                           :on_click="list_windows">
                 </k-button>
+      <k-accordion>
+        <k-accordion-item title = "Create Maintenance Window">
+                <k-input placeholder = "Id" :value.sync ="id"></k-input>
+                <k-input placeholder = "Description" :value.sync = "description"></k-input>
+                <k-input placeholder = "Start time" :value.sync = "start"></k-input>
+                <k-input placeholder = "End time" :value.sync = "end"></k-input>
+                <k-select icon = "link" title = "List of Switches" :options = "dpid_names" :value.sync = "items"></k-select>
+                <k-select icon = "link" title = "List of Interfaces" :options = "interface_names" :value.sync = "interface_items"></k-select>
+                <k-select icon = "link" title = "List of Links" :options = "link_names" :value.sync = "link_items"></k-select>
+                <k-input placeholder = "Status, Input 0, 1, or 2" :value.sync = "status" ></k-input>
```

Comment:
@rohanCodes001 could you please remove this `status` property being set, and also remove it from the request? It turns out that this shouldn't set when creating maintenance windows, the backend should be validating this, it's a validation issue #48 .